routers/img.js

```diff
@@ -210,7 +217,7 @@ router.get("/:id/info", async (req, res) => {
     }
 });
 
-router.get("/:id", async (req, res) => {
+router.get("/:id", fetch_rate_limit, async (req, res) => {
```

Comment:
Added a simple rate limit to image fetching to slow down the usage of Firebase bandwidth.